v0/util/index.js

```diff
@@ -953,6 +953,14 @@ function returnArrayOfSubarrays(arr, len) {
   return chunks;
 }
 
+// Helper method to add external Id to traits
+// Traverse through the possible keys for traits using generic mapping and add externalId if traits found
+function addExternalIdToTraits(message) {
+    const identifierType = message.context.externalId[0].identifierType;
```

Comment:
According to our previous schema, `externalId` objects have fields `type` and `id`. Are we changing this purposefully? If not, we should keep it same.

Reply:
`type` will be the Object type where as `identifierType` is the actual unique field  name and the `id` contains the value for that unique field. So, we just add the field and the value to traits.